routes/auth.js

```diff
@@ -1,7 +1,10 @@
 const express = require("express");
 const router = express.Router();
 const auth = require("../controllers/auth");
+const signout = require("../controllers/signout");
```

Comment:
Do we need to make a separate controller for signout or we can add the controller method in auth controller?

Reply:
Ok will do that